google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/datacatalog_entry_factory.py

```diff
@@ -145,7 +145,37 @@ def make_entry_for_dimension(self, dimension_metadata):
         # jump directly to a Dimension 'edit' page in Qlik Sense.
 
         # The create_time and update_time fields are not fulfilled because
-        # there is no such info in the Dimension properties.
+        # there is no such info in the Dimension metadata.
+
+        return generated_id, entry
+
+    def make_entry_for_measure(self, measure_metadata):
+        entry = datacatalog.Entry()
+
+        app_metadata = measure_metadata.get('app')
+
+        # The Measure ID is usually a 7 letters string, so the App ID is
+        # prepended to prevent overlaping.
```

Comment:
typo here: `overlaping` -> `overlapping`

I remember that the other master item method was this same comment, please check there too.

Reply:
Done, thanks!